### ISSUE TITLE
windows builds failing during autobuild

### ISSUE THREAD

**Issue by nitbhat** [UIUC-PPL/charm#2674]
Autobuild link: http://charm.cs.illinois.edu/autobuild/old.2020_01_13__01_02/mpi-win-x86_64.txt

This is possibly because of a build failure while compiling charmrun.

```
make[1]: Leaving directory '/home/nikhil/autobuild/netlrts-win-x86_64-smp/charm/netlrts-win-x86_64-smp/tmp/charmrun-src'
make[1]: Target 'all' not remade because of errors.
make: *** [Makefile.machine:2: charmrun] Error 2
```

**Comment by evan-charmworks** [comment]
There is also the `putenv` issue that was found as part of #2288

**Comment by matthiasdiener** [comment]
> There is also the `putenv` issue that was found as part of #2288

The putenv issue comes from #2632 which (I believe) wasn't merged to 6.10, only 6.11. So it could remain as part of #2288 I think.
